build.gradle

```diff
@@ -37,6 +37,7 @@ repositories {
 java {
     toolchain {
         languageVersion = JavaLanguageVersion.of(8)
+       // languageVersion = JavaLanguageVersion.of(17)
```

Comment:
remove ?

Reply:
Maybe add // for debug run with that - languageVersion = JavaLanguageVersion.of(17)
weird i know but it failed with java8